webcomponents/input/input.webcomponent.spec.js

```diff
@@ -183,6 +183,16 @@ describe('mn-input (webcomponent)', () => {
   })
 
   describe('attribute readonly', () => {
+    it('should define attribute inside of a element', () => {
+      element.setAttribute('readonly','readonly')                     // even no setting value, the attribute readonly is receiving value readonly as value.
```

Comment:
Even i commenting this line, the readonly attribute is receiving value and it can't verify if exist attribute readonly inside of input